hawkular-alerts-engine/src/test/java/org/hawkular/alerts/engine/CassPersistenceTest.java

```diff
@@ -61,14 +61,7 @@ public static void initSessionAndResetTestSchema() throws Exception {
 
         keyspace = AlertProperties.getProperty("hawkular-alerts.cassandra-keyspace", "hawkular_alerts_test");
 
-        Session session = CassCluster.getSession();
-
-        // try and clean up if this was somehow left around by a prior run
-        try {
-            session.execute("DROP KEYSPACE " + keyspace);
-        } catch (Throwable t) {
-            // never mind, it may not be there
-        }
+        Session session = CassCluster.getSession(true);
```

Comment:
It used to be that we would drop hawkular_alerts_test keyspace by default, is that still true?  It seems now that we must have hawkular-alerts.cassandra-keyspace set to "hawkular_alerts_test". Where is that done?

Perhaps maybe we want  CassCluster.getCleanSession(String keyspace) or something like that?